tests/bumper-spec.js

```diff
@@ -609,65 +639,134 @@ describe('Bumper', function () {
+          expect(bumper.vcs.getPr).to.have.been.calledWith('30')
+        })
+
+        it('should reject with the error', function () {
+          expect(rejection).to.equal('the-error')
+        })
+      })
     })
 
-    describe('when getPr succeeds', function () {
+    describe('bitbucket git log', function () {
       beforeEach(function () {
-        getPrResolver.resolve('the-pr')
-        return promise
+        // bitbucket merges PRs differently
+        const gitLog =
+          '98a148c Added some more tests, just a few more to go\n' +
+          '1b1bd97 Added some real unit tests\n' +
+          '10b394e Merged in feature/branch-name (pull request #300)' +
+          'fa066f2 Removed newline from parsed PR number\n' +
+          'fc416cc Automated version bump\n' +
+          '67db358 Fix for #26 by reading PR # from git commit\n' +
+          '4a61a20 Automated version bump\n' +
+          '7db44e1 Automated version bump\n' +
+          'f571451 add pullapprove config\n' +
+          '4398a26 address PR concerns\n'
+
+        execStub.returns(Promise.resolve(gitLog))
+
+        promise = bumper._getLastPr()
+          .then((pr) => {
+            resolution = pr
+            return pr
+          })
+          .catch((err) => {
+            rejection = err
+            throw err
+          })
       })
 
-      it('should parse out the PR number from the git log and passes it to vcs.getPr()', function () {
-        expect(bumper.vcs.getPr).to.have.been.calledWith('30')
+      it('should call git log', function () {
+        expect(execStub).to.have.been.calledWith('git log -10 --oneline')
       })
 
-      it('should resolve with the pr', function () {
-        expect(resolution).to.equal('the-pr')
-      })
-    })
+      describe('when getPr succeeds', function () {
+        beforeEach(function () {
+          getPrResolver.resolve('the-pr')
+          return promise
+        })
 
-    describe('when getPr fails', function () {
-      beforeEach(function (done) {
-        getPrResolver.reject('the-error')
-        promise.catch(() => {
-          done()
+        it('should parse out the PR number from the git log and passes it to vcs.getPr()', function () {
+          expect(bumper.vcs.getPr).to.have.been.calledWith('300')
         })
-      })
 
-      it('should parse out the PR number from the git log and passes it to vcs.getPr()', function () {
-        expect(bumper.vcs.getPr).to.have.been.calledWith('30')
+        it('should resolve with the pr', function () {
+          expect(resolution).to.equal('the-pr')
+        })
       })
 
-      it('should reject with the error', function () {
-        expect(rejection).to.equal('the-error')
+      describe('when getPr fails', function () {
+        beforeEach(function (done) {
+          getPrResolver.reject('the-error')
+          promise.catch(() => {
+            done()
+          })
```

Comment:
You can actually just return the `.catch()` of the promise in `beforeEach` and not worry about `done()` as well:

```js
beforeEach(function() {
  getPrResolver.reject('the-error')
  return promise.catch(() => {})
})
```

Just in case you want to type a little less ;)

Reply:
Updated